data/mods/Magiclysm/Spells/attunements/Permafrost_Mage.json

```diff
@@ -38,5 +38,42 @@
     "material": [ "magical_material" ],
     "flags": [ "AURA", "ONLY_ONE", "TRADER_AVOID", "NO_TAKEOFF", "NONCONDUCTIVE", "OVERSIZE" ],
     "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "emitter": "emit_cold_energy_strong" } ] }
+  },
+  {
+    "id": "permafrost_mage_ice_age",
+    "type": "SPELL",
+    "name": "Ice Age",
+    "description": "Cool down every creature in range till their flesh will crack, bones will shatter, and armor will fracture.",
+    "effect": "attack",
+    "effect_str": "cold",
+    "shape": "blast",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "NO_LEGS", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "max_level": 35,
+    "min_damage": 0,
+    "max_damage": 100,
+    "damage_increment": 3,
+    "min_dot": 0,
+    "max_dot": 45,
+    "dot_increment": 1.3,
+    "min_aoe": 3,
+    "max_aoe": 9,
+    "aoe_increment": 0.2,
+    "min_range": 5,
+    "max_range": 40,
+    "range_increment": 1,
+    "spell_class": "PERMAFROST_MAGE",
+    "base_casting_time": 500,
+    "casting_time_increment": -15,
+    "final_casting_time": 100,
+    "base_energy_cost": 700,
+    "energy_increment": -15,
+    "final_energy_cost": 300,
+    "min_duration": 500,
+    "max_duration": 18000,
```

Comment:
So is this 300 minutes for max duration?  or 1.8 MILLION hp damage?  Does it affect the area and anything walking through it during that time or just the monster it hit? I'm assuming just monsters it hits.

Reply:
No, its just 3 minute on max lvl, spell duration writet in, eeh, microtics? (ie 100 means 1 second, it's maded to work with spells that spend, for example, half of a second, or quarter of a second)